src/encoded/tests/workbook_fixtures.py

```diff
@@ -67,6 +72,8 @@ def workbook(app):
     if load_res:
         raise(load_res)
 
+    time.sleep(3)  # Fixture is time-consuming anyway, so indulge a sleep to let SQS internals to catch up
```

Comment:
This is a small but non-cosmetic change. Since this fixture is created only once, it shouldn't add much to overall testing.